modules/conformance-pack/main.tf

```diff
@@ -1,5 +1,5 @@
-resource "aws_config_conformance_pack" "hipaa" {
-  name = "operational-best-practices-for-HIPAA-Security"
+resource "aws_config_conformance_pack" "default" {
```

Comment:
would this rename `hipaa` => `default` cause issues when deleting and recreating?

Reply:
well, true, but I don't like leaving such a confusing resource name as well

Reply:
Yeah, this was a typo from the original, so no problem with the resource name change.

Reply:
Seems like most places we've standardized on `this` vs `default`, but I don't think I have enough empirical evidence on this to make a strong suggestion.